ajax/libs/jquery-nice-select/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "jquery-nice-select",
+  "filename": "jquery.nice-select.min.js",
+  "version": "1.1.0",
+  "description": "A lightweight jQuery plugin that replaces native select elements with customizable dropdowns.",
+  "keywords": [
+    "jquery",
+    "dropdown"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/hernansartorio/jquery-nice-select.git"
+  },
+  "author": {
+    "name": "Hernán Sartorio",
+    "url": "https://github.com/hernansartorio"
+  },
+  "license": "ISC",
+  "homepage": "http://hernansartorio.com/jquery-nice-select/",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/hernansartorio/jquery-nice-select.git",
+    "fileMap": [
+      {
+        "basePath": "js",
+        "files": [
+          "**/jquery.nice-select.*"
+        ]
+      },
+      {
+        "basePath": "css",
+        "files": [
+          "**/*"
+        ]
+      }
```

Comment:
@hare1039 
1. I don't think we need `style.css` 
2. it's a little bit not safe or no sense to use `**/*` under a non-dist folder. 
3. It's not good to use `js` and `css` as basePath.  

Let's leave basePath blank and combine these two using `+(js|css)/*nice-select*`